From da5f3fb5fdf605578fd5afc408630ead6c1fe5e3 Mon Sep 17 00:00:00 2001
From: Vaibhav Bedia <vaibhav.bedia@ti.com>
Date: Thu, 11 Nov 2010 17:12:59 +0530
Subject: [PATCH 2/3] ti816x: Default build for DDR2 @ 400MHz I/O clock

The default configuration will be for DDR2 with the
I/O clock @ 400MHz

Signed-off-by: Vaibhav Bedia <vaibhav.bedia@ti.com>
---
 arch/arm/include/asm/arch-ti81xx/clocks_ti816x.h |    2 +-
 include/configs/ti8168_evm.h                     |    4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm/include/asm/arch-ti81xx/clocks_ti816x.h b/arch/arm/include/asm/arch-ti81xx/clocks_ti816x.h
index b8e9136..6c9e4f8 100644
--- a/arch/arm/include/asm/arch-ti81xx/clocks_ti816x.h
+++ b/arch/arm/include/asm/arch-ti81xx/clocks_ti816x.h
@@ -29,7 +29,7 @@
 #define FAPLL_K			8
 #define SYSCLK_2_DIV		1
 #define OSC_FREQ		27
-#define DDR_PLL_796	/* Values supported 400,531,675,796 */
+#define DDR_PLL_400	/* Values supported 400,531,675,796 */
 
 /* Main PLL */
 #define MAIN_N			64
diff --git a/include/configs/ti8168_evm.h b/include/configs/ti8168_evm.h
index ccaf9dd..4f41e51 100644
--- a/include/configs/ti8168_evm.h
+++ b/include/configs/ti8168_evm.h
@@ -33,8 +33,8 @@
 						   initial data */
 
 /* Only one the following two options (DDR3/DDR2) should be enabled */
-#define CONFIG_TI816X_EVM_DDR3			/* Configure DDR3 in U-Boot */
-//#define CONFIG_TI816X_EVM_DDR2				/* Configure DDR2 in U-Boot */
+//#define CONFIG_TI816X_EVM_DDR3			/* Configure DDR3 in U-Boot */
+#define CONFIG_TI816X_EVM_DDR2				/* Configure DDR2 in U-Boot */
 #define CONFIG_TI816X_TWO_EMIF		1
 #define CONFIG_MISC_INIT_R		1
 #define CONFIG_TI816X_ASCIIART		1	/* The eye */
-- 
1.6.2.4

